packages/docs/community/release-process/smart-contracts.md

```diff
@@ -128,7 +131,9 @@ Deploying a new contract release should occur with the following process:
     <td>
       <ol>
         <li>Confirm all contracts working as intended on Alfajores.</li>
+        <li>Confirm audit is complete and make sure your forum post contains a link to it.</li>
```

Comment:
We should probably provide guidelines on security audits to perform and which auditors have worked well in the past. This isn't needed for now but probably for a future PR.

A lot of new dapps show they're audited and use it as a marketing scheme as a badge of honor without providing a lot of information on the auditors. Other times, they ask for an audit a few days before going live, not leaving enough time for professionals to properly audit the smart contract changes.